selfdrive/debug/print_docs_diff.py

```diff
@@ -1,120 +1,93 @@
+  for header in headers:
+    if new_car and new_car.get(header) != base_car.get(header):
+      row_builder.append(f"{_preprocess_cell(base_car.get(header, ''))} {ARROW_SYMBOL} {_preprocess_cell(new_car.get(header, ''))}")
+    else:
+      row_builder.append(_preprocess_cell(base_car[header]))
+  return "|" + "|".join(row_builder) + "|"
 
-  changes = defaultdict(list)
-  for base_car_model, base_cars in base_car_docs.items():
-    # Match car info changes, and get additions and removals
-    new_cars = new_car_docs[base_car_model]
-    car_changes, car_additions, car_removals = match_cars(base_cars, new_cars)
 
-    # Removals
-    for car_docs in car_removals:
-      changes["removals"].append(format_row([car_docs.get_column(column, STAR_ICON, VIDEO_ICON, FOOTNOTE_TAG) for column in Column]))
+def print_car_docs_diff(new_docs_path):
+  base_docs_content = requests.get("https://raw.githubusercontent.com/commaai/openpilot/master/docs/CARS.md").text
 
-    # Additions
-    for car_docs in car_additions:
-      changes["additions"].append(format_row([car_docs.get_column(column, STAR_ICON, VIDEO_ICON, FOOTNOTE_TAG) for column in Column]))
+  with open(new_docs_path) as new_docs_file:
+    new_docs_content = new_docs_file.read()
 
-    for new_car, base_car in car_changes:
-      # Column changes
-      row_diff = build_column_diff(base_car, new_car)
-      if ARROW_SYMBOL in row_diff:
-        changes["column"].append(row_diff)
+  base_headers, base_cars = get_cars_docs_in_markdown(base_docs_content)
+  new_headers, new_cars = get_cars_docs_in_markdown(new_docs_content)
 
-      # Detail sentence changes
-      if base_car.detail_sentence != new_car.detail_sentence:
-        changes["detail"].append(f"- Sentence for {base_car.name} changed!\n" +
-                                 "  ```diff\n" +
-                                 f"  - {base_car.detail_sentence}\n" +
-                                 f"  + {new_car.detail_sentence}\n" +
-                                 "  ```")
+  changes = {
+    'column': [build_row(base_headers, base_car, new_car) for base_car, new_car in find_changed_cars(base_cars, new_cars)],
+    'removals': [build_row(base_headers, car) for car in find_removed_cars(base_cars, new_cars)],
+    'additions': [build_row(new_headers, car) for car in find_added_cars(base_cars, new_cars)],
+  }
 
   # Print diff
   if any(len(c) for c in changes.values()):
     markdown_builder = ["### ⚠️ This PR makes changes to [CARS.md](../blob/master/docs/CARS.md) ⚠️"]
 
-    for title, category in (("## 🔀 Column Changes", "column"), ("## ❌ Removed", "removals"),
-                            ("## ➕ Added", "additions"), ("## 📖 Detail Sentence Changes", "detail")):
-      if len(changes[category]):
+    for title, category in (("## 🔀 Column Changes", "column"), ("## ❌ Removed", "removals"), ("## ➕ Added", "additions")):
+      ordered_headers = base_headers if category != 'additions' else new_headers
+      max_rows_to_show = 10
```

Comment:
Unless there is a problem specific to this PR, let's keep this printing behavior the same (no row limit).